Update motion_blinds.markdown (#34736)
home-assistant/home-assistant.github.io: source/_integrations/motion_blinds.markdown

### Connector app

>  To get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)), follow these steps:

Reviewer: ```suggestion
 To get the API key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)), follow these steps:
```

 
  1. In the left sidebar of the app, open the **Settings** {% icon "mdi:gear-outline" %} (gear icon). 

>   2. Select the **About** page of the Connector app 5 times. 

Reviewer 2: ```suggestion
  2. Select the **About** page of the Connector app.
  3. Tap the screen 5 times while being on the **About** page. 
```

### Connector app

> Click the About page of the Connector app 5 times to get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)). The About page is in the left sidebar of the app, under Settings (gear icon). Selecting About opens the About page, which you can tap on 5 times to open a window with the API key.

Reviewer: ```suggestion
 To get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)), follow these steps:
 
  1. In the left sidebar of the app, open the **Settings** {% icon "mdi:gear-outline" %} (gear icon). 
  2. Select the **About** page of the Connector app 5 times. 
      - This opens a window with the API key.
```
